src/utilities/tests/SimpleTestModule.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
:small_red_triangle_down: Rather than move this, we should copy it. Tests should be as self contained as possible/reasonable. As a result, any change to this test module would affect the other test and vice-versa.

Reply:
Got it that makes sense.